pvlib/losses.py

```diff
@@ -85,3 +86,128 @@ def soiling_hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
     soiling_ratio = 1 - 0.3437 * erf(0.17 * accum_mass**0.8473)
 
     return soiling_ratio
+
+
+def soiling_kimber(rainfall_timeseries, threshold=6, soiling_rate=0.0015,
+                   grace_period=14, max_soiling=0.3, manual_wash_dates=None,
+                   initial_soiling=0):
+    """
+    Kimber soiling model [1]_ assumes soiling builds-up at a daily rate unless
+    the daily rainfall is greater than a threshold. The model also assumes that
+    if daily rainfall has exceeded the threshold within a grace period, then
+    the ground is too damp to cause soiling build-up. The model also assumes
+    there is a maximum soiling build-up. Scheduled manual washes and rain
+    events are assumed to reset soiling to zero.
+
+    Parameters
+    ----------
+    rainfall_timeseries : pandas.Series
+        a timeseries of rainfall in millimeters
+    threshold : float, default 6
+        the amount of rain in millimeters [mm] required to clean the panels
+    soiling_rate: float, default 0.0015
+        daily soiling rate, enter as fraction, not percent, default is 0.15%
+    grace_period : int, default 14
+        The number of days after a rainfall event when it's assumed the ground
+        is damp, and so it's assumed there is no soiling. Change to smaller
+        value for dry climate, default is 14-days
+    max_soiling : float, default 0.3
+        maximum soiling, soiling will build-up until this value, enter as
+        fraction, not percent, default is 30%
+    manual_wash_dates : sequence or None, default None
+        A list or tuple of Python ``datetime.date`` when the panels were
+        manually cleaned. Note there is no grace period after a manual
+        cleaning, so soiling begins to build-up immediately after a manual
+        cleaning
+    initial_soiling : float, default 0
+        the initial fraction of soiling on the panels at time zero in the input
+
+    Returns
+    -------
+    soiling : pandas.Series
+        soiling build-up fraction
```

Comment:
```suggestion
        fraction of daily energy lost to soiling. Series has intervals of one day.
```

Reply:
the output has the same intervals as the input, so if the input is in hours, the output is also hours

Reply:
Right, my mistake.